package.json

```diff
@@ -94,4 +94,4 @@
   "dependencies": {
     "dotnet-2.0.0": "^1.4.4"
   }
-}
+}
```

Comment:
We should probably discuss conventions around empty new line at the end of files because I think my settings keep adding them and yours keep removing ;)